src/java.sql.rowset/share/classes/com/sun/rowset/RowSetResourceBundle.properties

```diff
@@ -70,7 +70,7 @@ cachedrowsetimpl.unsetmatch1 = Use column name as argument to unsetMatchColumn
 cachedrowsetimpl.unsetmatch2 = Use column ID as argument to unsetMatchColumn
 cachedrowsetimpl.numrows = Number of rows is less than zero or less than fetch size
 cachedrowsetimpl.startpos = Start position cannot be negative
-cachedrowsetimpl.nextpage = Populate data before calling 
```

Comment:
This sounded like it could potentially be followed by a name, but this is not the case.